lib/ui/screen/main.dart

```diff
@@ -7,13 +7,23 @@ import 'package:flutter/material.dart';
 import 'package:flutter_slidable/flutter_slidable.dart';
 import 'package:font_awesome_flutter/font_awesome_flutter.dart';
 import 'package:alert_me/domain/mapper/TimeUtil.dart';
+import 'package:workmanager/workmanager.dart';
 import 'package:timezone/timezone.dart' as tz;
 import 'package:timezone/data/latest.dart' as tzl;
-
 import '../component/alert_filter_bar.dart';
 
-void main() {
+void callbackDispatcher() {
+  Workmanager().executeTask((task, inputData) async {
+    NotificationService().showNotification(title: inputData?["title"],body: inputData?["body"]);
```

Comment:
I think we need to include "id" as one of the inputData. Then we can perform DB operation in the callback, something like this
```dart
 var dbAlert = await AlarmDatabase.instance.readAlert(inputData?["id"]);
 AlarmDatabase.instance.update(dbAlert.copy(status: AlertStatus.triggered));
```

Reply:
@WenjiuWang addressed your comments